### PR TITLE
UID byte gen

### PR DESCRIPTION
**Pull request by busheezy**
I've added some UID byte gen to the docs. I also fixed a problem with the docker image.

Thanks.

### CODE REVIEW

File: docs/assets/javascripts/binding-phrase.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  bfOutput.textContent = `set expresslrs_uid = ${uidBytes}\nsave`;
+}
+
+function updateValue(e) {
+  setOutput(e.target.value);
+}
+
+window.addEventListener("load", (event) => {
+  const input = document.querySelector(".bp-input");
+  input.addEventListener("input", updateValue);
```
Comment (jurgelenas) [inline docs/assets/javascripts/binding-phrase.js:40]: Consider checking is input variable is defined

File: docs/hardware/spi-receivers.md [hunk 2]
```diff
@@ -30,25 +30,48 @@ Video Tutorial (thanks to @JyeSmith):
...
+```
+```
+
+#### Setting Binding Phrase
+Go to Betaflight CLI and enter the following commands. (Enter your binding phrase above)
+```
+```
+
+<script type="text/javascript" src="//unpkg.com/crypto-js@4.1.1/crypto-js.js"></script>
+<script type="text/javascript" src="/assets/javascripts/binding-phrase.js"></script>
```
Comment (jurgelenas) [inline docs/hardware/spi-receivers.md:65]: I am not sure if absolute javascript file import will work together with mkdocs versioning. You might need to use `extra_javascript` https://squidfunk.github.io/mkdocs-material/customization/#additional-javascript

File: docs/assets/javascripts/binding-phrase.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+  const bindingPhraseHashed = CryptoJS.MD5(bindingPhraseFullEncoded);
+  const uidBytes = getBytesFromWordArray(bindingPhraseHashed);
+
+  return uidBytes;
+}
+
+function setOutput(text) {
+  const uidBytes = uidBytesFromText(text);
+  const output = document.querySelector("#__code_0 code");
```
Comment (jurgelenas) [inline docs/assets/javascripts/binding-phrase.js:27]: Is it going to break when additional code blocks are added to this page?

**Review by jurgelenas — CHANGES_REQUESTED** [review]

**Comment by jurgelenas** [comment]
Once finished, this will be awesome improvement for people with SPI receivers. Thank you @busheezy! :)

File: docs/assets/javascripts/binding-phrase.js (hunk 3 above)
Comment (busheezy) [reply docs/assets/javascripts/binding-phrase.js:27]: I have redone the element selection to only look for empty code blocks and to fail silently if they are not found.

**Comment by busheezy** [comment]
I'm not sure if I was supposed to mark those as resolved or not. If not, sorry. I have made some changes that should maybe take care of your concerns.

**Review by maybenikhil — APPROVED** [review]

**Review by jurgelenas — APPROVED** [review]

**Pull request merged**